src/sentry/tasks/data_export.py

```diff
@@ -143,9 +155,33 @@ def get_file_name(export_type, custom_string, extension="csv"):
     return file_name
 
 
-def alert_error():
-    # TODO(Leander): Handle errors in these tasks.
-    return
+# Adapted into contextmanager from 'src/sentry/api/endpoints/organization_events.py'
+@contextmanager
+def snuba_error_handler():
```

Comment:
This will handle any of the possible snuba errors and send them to sentry/email the user